This patch fix is required to port this file to solaris.
This patch has not been submitted upstream, but is intended to by
2015-Jan-31.
--- openscap-1.1.1/src/SCE/sce_engine.c.~1~	2014-10-27 14:47:43.904868342 -0700
+++ openscap-1.1.1/src/SCE/sce_engine.c	2014-10-27 14:49:29.343125585 -0700
@@ -45,7 +45,9 @@
 #include <assert.h>
 #include <fcntl.h>
 #include <sys/types.h>
+#if defined(__linux__)
 #include <sys/prctl.h>
+#endif
 #include <limits.h>
 #include <unistd.h>
 
